### DISCUSSION TITLE
Change the Toast banner

### DISCUSSION THREAD

**Discussion by phunkysc** [PSAppDeployToolkit/PSAppDeployToolkit#842] [Q&A] [answered]
Think I'm going crazy, but how do you change the banner text in the toast notifications to something other than Windows PowerShell? I can't seem to find it in the AppDeployToolkitConfig.xml file, unless it's somewhere else.

**Comment by AScott-WWF** [comment, marked as answer]
I think you may be looking for the `<!--Toast Notification Options-->` section
It should be something like this:
```
<!--Toast Notification Options-->
    <Toast_Options>
        <Toast_Disable>True</Toast_Disable>
        <Toast_AppName>PSAppDeployToolkit</Toast_AppName>
    </Toast_Options>
<!--Toast Notification Options-->
```
FYI: On an unmodified version 3.9.3 of the `AppDeployToolkitConfig.xml` file, this should be from line 66
